### PR TITLE
[DOCS] Update README, Remove Passive Voice

### PR DESCRIPTION
**Pull request by mikedelgaudio**
This is my initial PR for the Squoosh project before I dive into the homepage. 

## Goals:
- Removed passive voice from `README.md`
- Simplified the Developing section to copy and paste code snippets and signposting language
- Added a Contributing section to raise awareness for community contributions
- Modified introduction with inclusive language for all types of developer expertise
- Simplified API & CLI section with direct language
- Modified headings to gerunds when possible, providing a simplification between noun and verb

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
+   npm install
+   ```
+1. Then build the app by running:
+   ```sh
+   npm run build
+   ```
+1. After building, start the development server by running:
+   ```sh
+   npm run dev
+   ```
```
Comment (nucliweb) [inline README.md:36]: ### Add sequential number to have an order list

1. Clone the repository
2. To install node packages, run:
   ```sh
   npm install
   ```
3. Then build the app by running:
   ```sh
   npm run build
   ```
4. After building, start the development server by running:
   ```sh
   npm run dev
   ```

**Review by nucliweb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mikedelgaudio) [reply README.md:36]: Hi @nucliweb, 

Please see https://github.com/mikedelgaudio/squoosh/tree/update-readme I believe markdown supports an automatic scalable number increase by using `1.` Also https://www.markdownguide.org/basic-syntax/#ordered-lists

Screen Shot 2021-07-12 at 2 36 23 PM

Comment (nucliweb) [reply README.md:36]: Ok, I think that is more semantic code with incremental numbers 😉
But it's fine

File: README.md [hunk 2]
```diff
@@ -1,36 +1,42 @@
 # [Squoosh]!
 
-[Squoosh] is an image compression web app that allows you to dive into the advanced options provided
-by various image compressors.
+[Squoosh] is an image compression web app that provides lossless image quality with a significant reduction to file size.
```
Comment (surma) [inline README.md]: “provides lossless image quality” doesn’t sit well with me. We do mostly lossy compression. 
Maybe we can find a phrasing around “lets you explore image formats” or something? Also Squoosh is very much targeted at web developers, so you can lean into that if you think it makes sense.

**Review by surma — CHANGES_REQUESTED** [review]
Lovely work, @mikedelgaudio!

I only want to work on the opener a bit with you, the rest is brilliant as-is.

**Review by surma — APPROVED** [review]

**Pull request merged**